<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_user_in_db</class>
 <widget class="QDialog" name="add_user_in_db">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>565</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Reference Specialty</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Добавление пользователя</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Потдеврдить</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Студент</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
